packages/solana-test-utils/typings/index.d.ts

```diff
@@ -0,0 +1,25 @@
+export {};
+
+declare global {
```

Comment:
Is this working when used in another package? Is it copying to build dir?

Reply:
I am not sure yet - it is referenced in package.json but this may need some iteration to get right - can only test after publishing